### ISSUE TITLE
Default tab list never is replaced!

### ISSUE THREAD

**Issue by reallydrained** [Crimix/ServerTabInfo#23]
I have tried everything but I just can't get it to work. Idk what the issue might be but I am using it on my 1.7.10 forge server and it doesn't seem to do anything. Are there any specific commands or buttons I need to press? I can't seem to get it to do anything at all.

**Comment by Crimix** [comment]
No, if both the server and the client has the od, then everything should be working.
How does the tab list look like, You did not post any pictures in this issue

**Comment by reallydrained** [comment]
> No, if both the server and the client has the od, then everything should be working. How does the tab list look like, You did not post any pictures in this issue

The tab list is literally just vanilla, I am running it as part of a mod pack (yes, both on server and client) but it never does anything. I do not have any other mods interfering with tab list (it is vanilla!) so I don't think it is anything like that. I would have included pictures if there was anything to include but it would just be the default tab list. Sorry for the late reply, I never saw this. I have put this mod in the client/server folder like 10 times total now and that is why I am finally asking about it because I just cannot get it to do ANYTHING.

I am pretty sure I even setup a fresh server/instance a week or two ago with just this one mod and it didn't work there either. I can try this again, I just am not sure what's going wrong. This is like the one mod that will not work (other mods error/crash on startup but this one just does nothing). If a log would help I could provide that, I could run a fresh server/instance with just this mod and show you. I will do anything to help solve this issue, just let me know what you need. Thanks for replying at all btw =D

**Comment by Crimix** [comment]
I will have to find some time to take a look at 1.7.10. can you tell me exactly what version of Forge you are running

**Comment by reallydrained** [comment]
> I will have to find some time to take a look at 1.7.10. can you tell me exactly what version of Forge you are running

Forge 10.13.4.1614 (latest/recommended)

I would love to get it working =D
